assets/src/models/detoursList.ts

```diff
@@ -1,6 +1,7 @@
 import { array, Infer, nullable, number, string, type } from "superstruct"
 
 export const SimpleDetour = type({
+  uuid: number(),
```

Comment:
question: do you feel strongly about naming this `uuid`?
we're not using UUID's at this point, so maybe we just call it `id`?

Reply:
totally! I'm... not sure why I leaned into this naming here.